de/deploy/signup_pythonanywhere.md

```diff
@@ -1,11 +1,15 @@
-Erstelle dir einen kostenfreien "Beginner"-Account auf PythonAnywhere:
+PythonAnywhere ist ein Dienst, mittels dem Python auf Servern "in der Cloud" ausgeführt werden kann. Wir werden ihn verwenden um unsere Seite zu hosten, live und im Internet.
```

Comment:
Added comma on crowdin "Wir werden ihn verwenden, um..."

Reply:
That's actually optional (erweiterter Infinitiv), but probably indeed better with the comma.